packages/admin/blocks-admin/src/blocks/common/AdminComponentButton.tsx

```diff
@@ -16,7 +16,7 @@ export const AdminComponentButton = ({ variant, size, ...buttonProps }: PropsWit
         <AdminComponentPaper disablePadding>
             <Button
                 fullWidth
-                color={variant === "primary" ? "primary" : "info"}
+                variant="textDark"
```

Comment:
As this component is a custom styled button it probably should keep using the MUI `Button` and it's previous variant logic to avoid breaking changes like the color from changing. 

<img width="497" alt="AdminComponentButton" src="https://github.com/user-attachments/assets/ce9bc6f6-08e5-453c-8b81-82cc7316de8a" />

<img width="491" alt="Add Block Button" src="https://github.com/user-attachments/assets/172257aa-b0ca-437f-99c0-91acfe00f5fb" />